### PR TITLE
changed module name to subprocess

### PR DESCRIPTION
**Pull request by shirishaganta**
To get rid of modulenotfound error

Signed-off-by: Shirisha Ganta <shiganta@in.ibm.com>

### CODE REVIEW

**Comment by shirishaganta** [comment]
Before:
======
JOB ID     : 64a793e8716dcd16314669569657d725bcb54163
JOB LOG    : /root/tests/results/job-2019-10-17T19.08-64a793e/job.log
 (1/1) tbench.py:tbench.test: ERROR: Traceback (most recent call last):\n  File "/usr/lib64/python3.6/imp.py", line 235, in load_module\n    return load_source(name, filename, file)\n  File "/usr/lib64/python3.6/imp.py", line 172, in load_source\n    module = _load(spec)\n  File "<frozen importli... (0.08 s)
RESULTS    : PASS 0 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 0.46 s
JOB HTML   : /root/tests/results/job-2019-10-17T19.08-64a793e/results.html

After:
======
JOB ID     : 27a5be038a6da3be7096fa8d268c00a6e2c08fb5
JOB LOG    : /root/tests/results/job-2019-10-17T20.52-27a5be0/job.log
 (1/1) tbench.py:tbench.test: \listen failed
PASS (6.78 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 7.15 s
JOB HTML   : /root/tests/results/job-2019-10-17T20.52-27a5be0/results.html

**Comment by PraveenPenguin** [comment]
@nasastry review please

File: perf/tbench.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def setUp(self):
     def test(self):
         # only supports combined server+client model at the moment
         # should support separate I suppose, but nobody uses it
-        nprocs = self.params.get('nprocs', default=commands.getoutput("nproc"))
+        nprocs = self.params.get('nprocs', default=subprocess.getoutput("nproc"))
```
Comment (harish-24) [inline perf/tbench.py:63]: @shirishaganta Can you import cpu util and use `cpu.online_count` or `cpu.total_count` whichever suits best?

Comment (shirishaganta) [reply perf/tbench.py]: okay

**Comment by shirishaganta** [comment]
[root@ltcfleet2-lp18 perf]# avocado run tbench.py
/usr/local/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
Fetching asset from tbench.py:tbench.test
JOB ID     : de8bb09ad4daa6298dec8a4bfa51d55d6bba7ae2
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2020-06-15T14.46-de8bb09/job.log
 (1/1) tbench.py:tbench.test: -waiting for connections
PASS (5.52 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2020-06-15T14.46-de8bb09/results.html
JOB TIME   : 5.57 s
[root@ltcfleet2-lp18 perf]#

**Comment by harish-24** [comment]
@shirishaganta Please re-base to master

File: perf/tbench.py [hunk 2]
```diff
@@ -69,7 +68,7 @@ def test(self):
             cmd = os.path.join(self.sourcedir, "tbench") + " " + args
             # Standard output is verbose and merely makes our debug logs huge
             # so we don't retain it.  It gets parsed for the results.
-            self.results = process.system_output(cmd, shell=True).decode()
+            self.results = process.system_output(cmd, shell=True).decode("utf-8")
```
Comment (harish-24) [inline perf/tbench.py:71]: This change is not necessary?

**Comment by harish-24** [comment]
@shirishaganta please post relevant log and it is good to go in.

**Comment by shirishaganta** [comment]
root@ltcfleet2-lp18 perf]# avocado run tbench.py
/usr/local/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
Fetching asset from tbench.py:tbench.test
JOB ID     : e5354982b4f1028900a268f98c011030015fa0e7
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2020-06-15T17.24-e535498/job.log
 (1/1) tbench.py:tbench.test: -waiting for connections
PASS (5.53 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2020-06-15T17.24-e535498/results.html
JOB TIME   : 5.58 s
[root@ltcfleet2-lp18 perf]#

**Pull request merged**